spec/queries/get_link_changes_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe Queries::GetLinkChanges do
+  describe '#as_json' do
```

Comment:
I think this needs to change to `as_hash` as well.

Reply:
Good spot, I've updated :)